safe_transaction_service/contracts/management/commands/update_safe_contracts_logo.py

```diff
@@ -7,6 +7,8 @@
 from config.settings.base import STATICFILES_DIRS
 from safe_transaction_service.contracts.models import Contract
 
+TRUSTED_FOR_DELEGATES = ["MultiSendCallOnly"]
```

Comment:
```suggestion
TRUSTED_FOR_DELEGATE_CALL = ["MultiSendCallOnly"]
```

Reply:
`TRUSTED_FOR_DELEGATES` is very ambiguous, more when we already have a lot of delegates logic that has nothing to do with delegate calls